thirdparty/cmdconfig/commands/fnevalcmd.go

```diff
@@ -65,6 +68,7 @@ type EvalFnRunner struct {
 	DryRun             bool
 	Image              string
 	ExecPath           string
+	FnConfig           string
```

Comment:
should we rename it to `FnConfigPath` to make it less ambiguous because `fnConfig` is so overloaded. We can keep the `commandline flag` to be `--fn-config`.

Reply:
Renamed this member.